### PR TITLE
[-] FO: Fix duplicate IDs in Link List module

### PR DESCRIPTION
**Pull request by martinfojtik**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | related to https://github.com/PrestaShop/ps_linklist/pull/3 |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |

### CODE REVIEW

**Comment by maximebiloe** [comment]
Thank you for contributing.

Waiting for https://github.com/PrestaShop/ps_linklist/pull/3 to be redone on the right branch.

**Comment by fojtcz** [comment]
done PrestaShop/ps_linklist#4

**Comment by mickaelandrieu** [comment]
@fojtcz would you mind to rebase this one against `develop` branch ?

**Comment by fojtcz** [comment]
@mickaelandrieu sure

File: themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl [hunk 1]
```diff
@@ -6,7 +6,7 @@
         {foreach $linkBlock.links as $link}
           <li>
             <a
-                id="{$link.id}"
+                id="{$link.id}-{$linkBlock.id"
```
Comment (Shudrum) [inline themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl]: Be careful, you missed the last `}`.

Comment (fojtcz) [inline themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl]: ohh, sorry..

**Comment by mickaelandrieu** [comment]
Thank you @fojtcz

**Pull request merged**